environments/prod/jcm-judiciary-uk.yml

```diff
@@ -6,7 +6,7 @@ A:
   - name: "@"
     ttl: 60
     alias_target_resource_id: "/subscriptions/8cbc6f36-7c56-4963-9d36-739db5d00b27/resourceGroups/lz-prod-rg/providers/Microsoft.Network/frontdoors/hmcts-prod"
-    shutter: false
+    shutter: true
```

Comment:
this ones shutter doesn't work and there's not much point changing it as it's just a redirect to XUI which will be shuttered as part of this.

Reply:
good to know.